templates/base/server/services/auth/service.py

```
@@ -0,0 +1,215 @@
+    TokenNotFoundException,
+    EmailNotVerifiedException,
+)
+from server.exceptions.user import UserNotCreatedException, UserNotFoundException
+from server.models import (
+    LoginRequest,
+    AuthResponse,
+    UserCreateRequest,
+    UserResponse,
+    UserUpdateRequest,
+    PasswordResetRequest,
+    LoginRequest,
```

Ruff (0.11.9)

Redefinition of unused `LoginRequest` from line 18

Remove definition: `LoginRequest`

(F811)